spec/controllers/teams_controller_spec.rb

```diff
@@ -60,9 +60,9 @@
     end
 
     context 'after acceptance letters have been sent' do
-      let(:last_season)      { Season.create name: Date.today.year - 1 }
-      let!(:voluntary_team)  { create :team, :in_current_season, kind: 'voluntary' }
-      let!(:sponsored_team)  { create :team, :in_current_season, kind: 'sponsored' }
+      let(:last_season) { Season.create name: Date.today.year - 1 }
+      let!(:deprecated_voluntary_team)  { create :team, :in_current_season, kind: 'deprecated_voluntary' }
```

Comment:
Same here: please remove anything related to the deprecated column and test for the new types of teams instead.